src/components/Dialog/Dialog.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import { GridCenter } from "../commonElement";
+import { Text } from "../Typography/Text/Text";
+import { Dialog } from "./Dialog";
+import Separator from "../Separator/Separator";
+import { Spacer } from "../Spacer/Spacer";
+import { Button } from "../Button/Button";
+import styled from "styled-components";
+import { Link } from "../Link/Link";
+
+const DialogComponent = ({
+  open,
+  title,
+  modal,
+  showClose,
+  forceMount,
+}: {
+  open: "default" | "open" | "closed";
+  title: string;
+  modal: boolean;
+  showClose: boolean;
+  forceMount?: boolean;
+}) => (
+  <GridCenter>
+    <Dialog
+      open={open === "default" ? undefined : open === "open"}
+      modal={modal}
+    >
+      <Dialog.Trigger>
+        <Link>Open dialog</Link>
+      </Dialog.Trigger>
```

Comment:
i think there are a few problems here. 

1. this bounds the trigger to always be defined inside a Dialog. What if the trigger and the content reside somewhere distant? That's a bit inconvenient but manageable.

2. It seems that there are some issues if we want the trigger to be a Click Ui button. Radix changes the style and the `type` property arbitrarily